server/app/models/fusor/deployment.rb

```diff
@@ -12,8 +12,20 @@
 
 module Fusor
   class Deployment < ActiveRecord::Base
-    serialize :rhev_params, Hash
-    serialize :cfme_params, Hash
-    serialize :openstack_params, Hash
+    belongs_to :organization
+    belongs_to :lifecycle_environment, :class_name => "Katello::KTEnvironment"
+
+    validates :name, :presence => true, :uniqueness => {:scope => :organization_id}
+    validates :organization_id, :presence => true
+    validates :lifecycle_environment_id, :presence => true
+
+    validates_with Validators::KatelloNameFormatValidator, :attributes => :name
+
+    # TODO: need to figure out the syntax for this
+    # has_one :host, foreign_key => :rhev_hypervisor_host_id, dependent => :nullify
+    # has_one :host, foreign_key => :rhev_engine_host_id, dependent => :nullify
+    #has_one :rhev_hypervisor_host_id
+    #has_one :rhev_engine_host_id
```

Comment:
deployment has_one engine, but can have many hypervisors. The first pass is only having one hypervisor, but from my understanding, we are supposedly to eventually support picking 2+ hypervisors. This can be in a separate PR though as I see this code is commented out.